lib/mutant/ast/named_children.rb

```diff
@@ -13,8 +13,6 @@ module NamedChildren
       # @api private
       #
       def self.included(host)
-        super
```

Comment:
I am actually fine with just saying that if you know what the parent class method is doing that you can decide to omit `super` if it does not break LSP. I don't think we should use `super` "just in case" the parent class decides to change it's interface.